docker/cache/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM debian:bullseye
-COPY ./distro/fuse-query /fuse-query
-COPY ./distro/fuse-store /fuse-store
+COPY ./distro/datafuse-query /datafuse-query
```

Comment:
renaming docker binaries should also update test-infra manifests

Reply:
https://github.com/datafuselabs/test-infra/pull/26